### PR TITLE
Improves rendered output of docs

### PR DESCRIPTION
**Pull request by cgranleese-r7**
Enhancement to https://github.com/rapid7/metasploit-framework/pull/17399

Makes some formatting changes to improve the rendered output of some docs that were added in a previous [PR](https://github.com/rapid7/metasploit-framework/pull/17399), specifically this [comment](https://github.com/rapid7/metasploit-framework/pull/17399#discussion_r1086041535).

As well as making some changes to add normal case mermaid labels see Alan's [comment](https://github.com/rapid7/metasploit-framework/pull/17537/files#r1086561956) for context.

## Verification

List the steps needed to make sure this thing works

- [ ] Follow these [docs](https://github.com/rapid7/metasploit-framework/tree/3beb3d775f7a9166dadfbbd9e392b443c51a7afd/docs#production-build) to build the docs site and ensure everything is formatted correctly
- [ ] Ensure CI passes

### CODE REVIEW

File: docs/_includes/header_custom.html [hunk 1]
```diff
@@ -2,4 +2,8 @@
 #main-content p {
     text-align: justify;
 }
+
+.language-mermaid .label {
+    text-transform: inherit;
+}
```
Comment (adfoster-r7) [inline docs/_includes/header_custom.html:8]: Context:

### Before

upper case mermaid labels

image

### After

normal case mermaid labels

image

We should get that fixed [upstream](https://github.com/just-the-docs/just-the-docs/blob/3d1f926a6821713598104aca6dbb28debca2a02f/_sass/code.scss#L204) - but not a blocker for now

**Comment by adfoster-r7** [comment]
Looks good to me

Before

image

After

image

**Comment by adfoster-r7** [comment]
## Release Notes

Improves the readability of the AD CS documentation

**Comment by gwillcox-r7** [comment]
Thanks for fixing this up @adfoster-r7 and @cgranleese-r7, much appreciated! ❤️

**Pull request merged**